content/applications/general/voip/axivox/vm_audio_messages.rst

```diff
@@ -0,0 +1,216 @@
+   this documentation: :doc:`axivox_config`.
+
+Upon connecting to the Axivox audio recorder management line a recorded French speaking operator
+provides the following options:
+
+#. Press `1` to record a message.
+#. Press `2` to listen to the current message.
+
+Press either `1` or `2` depending on whether there is already a message present in the system for
+this particular audio message (press `2`) or if a new audio message needs to be recorded (press
+`1`).
+
+Record the audio message after pressing `1`, then press `#` to end the recording. The French
+speaking operator returns to the line presenting the first set of questions again:
+
+#. Press `1` to record a message.
+#. Press `2` to listen to the current message.
+
+Press `#` to end the call.
+
+Write audio message
+-------------------
+
+The second option to obtain an audio message is to type the message out and select a computerized
+speaker to say the text. To do so, simply, navigate to the :guilabel:`Audio messages` page by
+clicking on :guilabel:`Audio messages` in the left panel. Then select the blue button labeled:
+:guilabel:`Text message`, next to the corresponding audio message :guilabel:`Name` that the audio
+message should be attached to.
+
+Click the drop-down menu next to the field labeled :guilabel:`Voice` and select a :guilabel:`Voice` for
+the :guilabel:`Text` to be read in. After the selection has been made, click :guilabel:`Generate` to
+process the audio file. The text is read in the language that it is written in the :guilabel:`Text`
+field. Should the language differ in the :guilabel:`Voice` then an accent is used by the
+computerized speaker.
+
+
+Finally when these steps are complete, click :guilabel:`Save` to save the audio message.
+
+To implement the changes, click :guilabel:`Apply changes` in the upper right-side of the screen.
+
+.. tip::
+   To use a audio message associated with a dial plan menu element see this documentation
+   :doc:`dial_plan_basics` or :doc:`dial_plan_advanced`.
+
+.. _axivox/music_on_hold:
+
+Music on-hold
+=============
+
+Axivox has the option to add custom hold music to the call whenever a caller is waiting for their
+call to be answered. To add *hold music* on to the Axivox administrative portal, navigate to the
+`manage.axivox.com <axivox_admin_>`_, and login. Click on :guilabel:`Music on hold` and a pop-up
+appears.
+
+On the :menuselection:`Change the music on hold` pop-up, click on the :guilabel:`Browse...` button
+to select the MP3 (MPEG Audio Layer 3) or WAV (Waveform Audio File Format ) file to be uploaded.
+
+.. note::
+   Only :abbr:`MP3 (MPEG Audio Layer 3)` or :abbr:`WAV (Waveform Audio File Format)` files can be
+   uploaded to the Axivox administrative portal.
+
+Once the file is selected, the :guilabel:`Progression` bar shows an upload status. When this
+activity completes, then the window can be closed, by clicking :guilabel:`Close`.
+
+:guilabel:`Apply changes` in the upper right-side of the screen.
```

Comment:
```suggestion
In Axivox, there are also numerous forwarding settings for a user. To access these forwardings
settings, go to `manage.axivox.com <axivox_admin_>`_ and log in.

Next, navigate to :menuselection:`Users`, located in the menu on the left.

From there, click into the specific user the forwarding should be added to. Then, open the
:guilabel:`Forwardings` tab.

If the user is busy on another call, or away from the phone, there is an option present in this tab
to :guilabel:`Send to voicemail as a last resort`, located in the :guilabel:`Forwarding on no
answer` and :guilabel:`Forwarding on busy` fields.

.. image:: vm_audio_messages/forwardings.png
   :align: center
   :alt: Send to voicemail as a last resort options highlighted on the Forwardings tab of the user.

If the :guilabel:`Send to voicemail as a last resort` box is ticked, when the forwarding actions
stated in each section are not successful, the caller is routed to the voicemail set on the
particular user.

.. seealso::
   For more information on forwarding and transfers visit :ref:`axivox/forwardings_tab`.

When all the desired configurations are complete, click :guilabel:`Save`, then click
:guilabel:`Apply changes` in the upper-right corner of the screen to implement the change.

Audio messages
==============

It is possible to add audio messages *before* a customer's call is even taken, to inform them about
the waiting time for deliveries, the availability of a product, or any other important promotional
messages.

To record an audio message in Axivox, navigate to `manage.axivox.com <axivox_admin_>`_ and log in.

Next, click on :guilabel:`Audio messages` in the menu on the left. From the :guilabel:`Audio
messages` page, click :guilabel:`Add a message`.

Type in a :guilabel:`Name`, and click :guilabel:`Save`.

Upon clicking :guilabel:`Save`, the browser redirects back to the main :guilabel:`Audio messages`
page, where the newly-created message can be found on the list.

There are two different ways to make the audio message. The user could either record the message
over the phone, or type the message (in text), and select a computer-generated speaker to read the
message.

Record audio message
--------------------

To record an audio message over the phone, click the orange button labeled,
:guilabel:`Record/Listen`, located to the right of the desired message on the list to record for, on
the :guilabel:`Audio messages` page.

When clicked, a :guilabel:`Record / listen to a message` pop-up window appears. From here, the
message is then recorded, via one of the extensions that is associated with the user.

Under :guilabel:`Extension to use for message management` field, click the drop-down menu, and
select the extension where Axivox should call to record the message.

Then, click :guilabel:`OK` to begin the call.

.. note::
   The user **must** be active in the production database with :abbr:`VoIP (Voice over Internet
   Protocol)` configured. To configure :abbr:`VoIP (Voice over Internet Protocol)` for a user, see
   this documentation: :doc:`axivox_config`.

Upon connecting to the Axivox audio recorder management line, a recorded French-speaking operator
provides the following options:

#. Press `1` to record a message.
#. Press `2` to listen to the current message.

Press either `1` or `2`, depending on whether or not there is already a message present in the
system for this particular audio message that requires a review, before recording a new one.

Record the new audio message after pressing `1`, then press `#` to end the recording.

The French-speaking operator returns to the line presenting the first set of questions again:

#. Press `1` to record a message.
#. Press `2` to listen to the current message.

Press `#` to end the call.

Write audio message
-------------------

To type the message, and select a computerized speaker to say the text, navigate to the
:menuselection:`Audio messages` in the menu on the left.

From the :guilabel:`Audio messages` page, select the blue button labeled, :guilabel:`Text message`,
next to the corresponding audio message :guilabel:`Name` that the message should be attached to.

Doing so reveals :guilabel:`Convert text to message` pop-up window.

From the :guilabel:`Convert to text message` pop-up window, click the drop-down menu next to the
field labeled, :guilabel:`Voice`, and select an option for the :guilabel:`Text` to be read in.

After the :guilabel:`Voice` selection has been made, and the message has been written in the
:guilabel:`Text` field, click :guilabel:`Generate` to process the audio file.

The text is read in the same language it is written in the :guilabel:`Text` field. Should the
language differ in the :guilabel:`Voice`, then an accent is used by the computerized speaker.

Finally, when these steps are complete, click :guilabel:`Save` to save the audio message.

To implement the changes, click :guilabel:`Apply changes` in the upper-right corner of the screen.

.. tip::
   To use an audio message associated with a dial plan menu element, see this documentation
   :doc:`dial_plan_basics` or :doc:`dial_plan_advanced`.

.. _axivox/music_on_hold:

Music on-hold
=============

Axivox has the option to add custom hold music to the call whenever a caller is waiting for their
call to be answered. To add hold music on to the Axivox administrative portal, navigate to the
`manage.axivox.com <axivox_admin_>`_, and log in.

Then, click on :guilabel:`Music on hold` from the menu on the left, and a :guilabel:`Change the
music on hold` pop-up window appears.

On the :menuselection:`Change the music on hold` pop-up window, click the :guilabel:`Choose File`
button to select an MP3 (MPEG Audio Layer 3) or WAV (Waveform Audio File Format ) file to be
uploaded.

.. note::
   Only :abbr:`MP3 (MPEG Audio Layer 3)` or :abbr:`WAV (Waveform Audio File Format)` files can be
   uploaded to the Axivox administrative portal.

Once the file is selected, the :guilabel:`Progression` bar shows an upload status. When this
activity completes, the window can be closed, by clicking :guilabel:`Close`.

When the desired changes are complete, click :guilabel:`Apply changes` in the upper-right corner of
the screen.
```
this doc might be even more beneficial with a few more screenshots, but that's just a suggestion